modules/seqexec/server/src/main/scala/seqexec/server/tcs/Tcs.scala

```diff
@@ -167,9 +167,9 @@ object Tcs {
     val gwp2 = config.extractAs[StandardGuideOptions.Value](TELESCOPE_KEY / GUIDE_WITH_PWFS2_PROP).toOption
     val gwoi = config.extractAs[StandardGuideOptions.Value](TELESCOPE_KEY / GUIDE_WITH_OIWFS_PROP).toOption
     val gwao = config.extractAs[StandardGuideOptions.Value](TELESCOPE_KEY / GUIDE_WITH_AOWFS_PROP).toOption
-    val offsetp = config.extractAs[Double](TELESCOPE_KEY / P_OFFSET_PROP).toOption
+    val offsetp = config.extractAs[String](TELESCOPE_KEY / P_OFFSET_PROP).toOption.flatMap(_.parseDoubleOption)
       .map(Arcseconds(_):Angle).map(tag[OffsetP](_))
-    val offsetq = config.extractAs[Double](TELESCOPE_KEY / Q_OFFSET_PROP).toOption
+    val offsetq = config.extractAs[String](TELESCOPE_KEY / Q_OFFSET_PROP).toOption.flatMap(_.parseDoubleOption)
```

Comment:
Shouldn't the offsets be sent as Double ?

Reply:
They should!